app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,52 @@
+class AlbumsController < ApplicationController
+
+  def index
+    @albums = Album.all
+  end
+
+  def new
+    @album = Album.new
+  end
+
+  def create
+    @album = Album.new(album_params)
+    if @album.save
+      redirect_to action: 'index'
```

Comment:
As best I can tell from this page (http://api.rubyonrails.org/classes/ActionController/Redirecting.html), this:
```
redirect_to action: 'index'
```
 does the same thing as 
```
redirect_to root_path
```
Both are a 302 redirect.